Makefile

```diff
@@ -77,11 +81,19 @@ clean:
 	cargo clean
 	go clean ./...
 
+# DEPRECATED - please use build-stellar-rpc instead
```

Comment:
Can you make sure we aren't referencing this anywhere if it's deprecated?

Reply:
it is soon to be deprecated actually, we will deprecate it fully after p22 just to reduce the chaos. Also can't do it now as we are still planning to issue 2 binaries [`soroban-rpc, stellar-rpc`]